clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/UpsertAlarmDescription.groovy

```diff
@@ -47,4 +48,24 @@ class UpsertAlarmDescription extends AbstractAmazonCredentialsDescription {
   Collection<String> alarmActionArns
   Collection<String> insufficientDataActionArns
   Collection<String> okActionArns
+
+  PutMetricAlarmRequest buildRequest() {
+    new PutMetricAlarmRequest(
+        alarmName: name,
+        actionsEnabled: actionsEnabled,
+        alarmDescription: alarmDescription,
+        comparisonOperator: comparisonOperator,
+        evaluationPeriods: evaluationPeriods,
+        period: period,
+        threshold: threshold,
+        namespace: namespace,
+        metricName: metricName,
+        statistic: statistic,
+        unit: unit,
+        dimensions: dimensions,
+        alarmActions: alarmActionArns,
+        insufficientDataActions: insufficientDataActionArns,
+        oKActions: okActionArns
+    )
+  }
```

Comment:
This seemed as good a place as any to consolidate this logic, since we're using it in two places now...